AndroidManifest.xml

```diff
@@ -73,7 +73,9 @@
         <activity android:name=".ui.activity.UploadFilesActivity" />
         <activity android:name=".ui.activity.ReceiveExternalFilesActivity"
                   android:taskAffinity=""
-                  android:excludeFromRecents="true">
+                  android:excludeFromRecents="true"
+                  android:label="@string/uploader_top_message"
```

Comment:
Better remove it, so that default "ownCloud" appears in the menu to choose sharee app.